web/client/plugins/CreateNewMap.jsx

```diff
@@ -17,14 +17,17 @@ const CreateNewMap = React.createClass({
         mapType: React.PropTypes.string,
         onGoToMap: React.PropTypes.func,
         colProps: React.PropTypes.object,
-        isLoggedIn: React.PropTypes.bool
+        isLoggedIn: React.PropTypes.bool,
+        allowedRoles: React.PropTypes.array,
+        user: React.PropTypes.object
     },
     contextTypes: {
         router: React.PropTypes.object
     },
     getDefaultProps() {
         return {
             mapType: "leaflet",
+            allowedRoles: ["ADMIN", "USER"],
```

Comment:
By default all logged in, but we can deny for normal users via configuration.Using groups is a good idea for the future.